src/test/unit/installer.test.ts

```diff
@@ -0,0 +1,49 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck ignore type checking in test files
+import { installTerraformLS } from '../../installer/installer';
+import { reporter } from '../../__mocks__/reporter';
+import { pathExists } from '../../installer/detector';
+import { window } from '../../__mocks__/vscode';
+
+jest.mock('../../installer/detector');
+describe('ls installer', () => {
+  describe('should install', () => {
+    let temp;
+
+    beforeEach(() => {
+      temp = require('temp').track();
+    });
+
+    test('when valid', async () => {
+      const installPath = temp.path('foo');
```

Comment:
We're mocking everything related to `installPath` / fs, so it should be safe to use a string here and get rid of `temp`